### ISSUE TITLE
errors but not sure if actually causing issue

### ISSUE THREAD

**Issue by amlwwalker** [therecipe/qt#175]
Got everything building, but during setup, and when running examples, I get:

```
ERRO[0002] failed to load /usr/local/opt/qt5/lib/QtGui.framework/Versions/5/Headers/5.7.0/QtGui/qpa/qplatformsystemtrayicon.h  error="open /usr/local/opt/qt5/lib/QtGui.framework/Versions/5/Headers/5.7.0/QtGui/qpa/qplatformsystemtrayicon.h: no such file or directory"
ERRO[0002] failed to load /usr/local/opt/qt5/lib/QtGui.framework/Versions/5/Headers/5.7.0/QtGui/qpa/qplatformgraphicsbuffer.h  error="open /usr/local/opt/qt5/lib/QtGui.framework/Versions/5/Headers/5.7.0/QtGui/qpa/qplatformgraphicsbuffer.h: no such file or directory"
```

every time.

On Mac OSX El Capitan, Golang 1.7.1

**Comment by therecipe** [comment]
Hey

Yes, these errors are expected and nothing to worry about.
They are only visible now, because I made `qtdeploy` a bit more verbose.
But `qtdeploy` will be quiet again soon.

**Comment by akamensky** [comment]
@therecipe I actually found it very useful, because it helped me to trace issue when `QT_VERSION` was not defined and therefore it was using current `QT_DIR` but inside it still was trying to use default `5.7.0` as reference. I think so verbosity CLI argument would be good to be able to get those errors.

**Comment by therecipe** [comment]
@akamensky Yes, no worry I'm not planning to hide all errors. Only those that are expected.
Also you should be able to make them visible again with the `-debug` flag since https://github.com/therecipe/qt/commit/7f3481d85f4349dfa06beb76efe1e51f101f788b, which should work with all `qt*` tools.
